Documentation/supermuc.rst

```diff
@@ -196,3 +125,170 @@ Many post-processing scripts of SeisSol require Python dependencies.
 We describe how to use pip on SuperMUC at see :ref:`pypi_behind_firewall`.
 
 
+Using Sanitizer
+---------------
+
+Note that to use sanitizer (https://en.wikipedia.org/wiki/AddressSanitizer), SeisSol needs to be compiled with gcc.
+For that modules and compiler need to be switched:
+
+.. code-block:: bash
+
+    module switch seissol-env seissol-env/develop-gcc11
+
+    export CC=mpigcc
+    export CXX=mpigxx
+    export FC=mpifc
+
+Then cmake on a new build folder.
+To enable sanitizer, add ``-DADDRESS_SANITIZER_DEBUG=ON`` to the argument list of cmake, and change the ``CMAKE_BUILD_TYPE`` to ``RelWithDebInfo`` or ``Debug``.
+
+
+Compiling seissol-env spack package
+-----------------------------------
+For reference, to compile seissol-env on SuperMUC-NG, follow the procedure below:
+
+.. code-block:: bash
+
+    # load spack
+    module load user_spack
+    # clone seissol-spack-aid and add the repository
+    # we use a supermuc specific branch as supermuc spack is too old (0.17.1) for the main branch
+    git clone --branch supermuc_NG https://github.com/SeisSol/seissol-spack-aid.git
+    cd seissol-spack-aid
+    spack repo add ./spack
+    # locate externally build pkg-config
+    spack external find pkg-config
+    # install all dependencies of seissol.
+    # We specify the intel and intel-mpi version matching preinstalled version on supermuc-ng
+    # These can be found with:
+    # >spack find intel-mpi
+    # >spack compiler list
+    spack install seissol-env %intel ^intel-mpi
+    # or
+    spack install seissol-env %gcc ^intel-mpi
+    # now create a module:
+    spack module tcl refresh seissol-env@develop%intel
+    #to access the module at start up, add to your ~/.bashrc
+    module use $HOME/spack/modules/x86_avx512/linux-sles15-skylake_avx512/
+    # change this path to your_custom_path_2_modules if you update ~/.spack/modules.yaml 
+
+Custom install directory for packages and modules can be set by changing ``~/.spack/config.yaml``
+
+.. code-block:: yaml
+
+    config:
+      install_tree: path_2_packages
+
+and ``~/.spack/modules.yaml``:
+
+.. code-block:: yaml
+
+    modules:
+      default:
+        roots:
+         tcl: your_custom_path_2_modules
+
+This can be useful to share packages with other users of a SuperMUC project.
+
+The seissol-env compilation can also be reduced by adding the python module to ``~/.spack/packages.yaml``:
+
+.. code-block:: yaml
+
+    packages:
+      python:
+        externals:
+        - spec: python@3.8.11
+          buildable: False
+          modules:
+           - python/3.8.11-extended
+
+
+Compiling seissol spack package
+-------------------------------
+ 
+The seissol package is similar with the seissol-env package (it gathers all dependencies of seissol), but also compiles a specific version of seissol.
```

Comment:
`similar to`

Reply:
Also: `but also compiles a specific version of seissol itself` ... or something to highlight that SeisSol is included here